pkg/fuzz/dataformat/multipart.go

```diff
@@ -76,15 +76,28 @@ func (m *MultiPartForm) Encode(data KV) (string, error) {
 			return true
 		}
 
-		// Add field
-		if fw, err = w.CreateFormField(key); err != nil {
-			Itererr = err
-			return false
+		// Handle form field values - can be string or []string for duplicate fields
+		var values []string
+		switch v := value.(type) {
+		case string:
+			values = []string{v}
+		case []string:
+			values = v
+		default:
+			// Fallback: attempt string conversion
+			values = []string{fmt.Sprint(v)}
```

Comment:
it has a flaw. if `value` is non-string type, ex. `[]any{1, "test"}`, `[]int{1, 2}`, then it will convert the entire slice into a single literal string instead of creating 2 separate fields for `1` and `"test"`. this is incorrect.